modules/post-install-edge-aws-extend-machineset.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="post-install-edge-aws-extend-machineset"]
+= Creating a machine set manifest for an AWS Local Zones node
+
+After you create subnets in AWS Local Zones, you can create a machine set manifest.
+
+The installation program sets the following labels for the `edge` machine pools at cluster installation time:
+
+* `machine.openshift.io/parent-zone-name: <value_of_ParentZoneName>`
+* `machine.openshift.io/zone-group: <value_of_ZoneGroup>`
+* `machine.openshift.io/zone-type: <value_of_ZoneType>`
```

Comment:
Not going to block merge on this, but typically we would force-lowercase any user-replaceable values (so, these, and any other similar instances throughout the PR, like `<value_of_GroupName>`), though I realize they're likely referring to the literal capitalization of the related fields.

Reply:
Thanks. That is correct given the nature of the literals and trying to be more direct so no confusion arises. Sometimes customer feedback comes in the form of changing these values , so using literals avoids this confusion.